src/strategies/tokenlon/index.ts

```diff
@@ -0,0 +1,54 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'TokenlonDev';
+export const version = '0.1.0';
+
+const TFT_ADDRESS = '0xE534619dEFDBF0cAf673b8AbF7158714F5BD4bd9';
```

Comment:
Can you make this a parameter in the strategy?

Reply:
Done.